### PR TITLE
Updated service.py

### PR DESCRIPTION
**Pull request by sg5717**
updated code to delete a product

### CODE REVIEW

**Comment by sg5717** [comment]
Issue #39

File: app/service.py [hunk 1]
```diff
@@ -186,19 +186,19 @@ def update_pets(pet_id):
 
 
 ######################################################################
-# DELETE A PET
+# DELETE A PRODUCT
 ######################################################################
-@app.route('/pets/<int:pet_id>', methods=['DELETE'])
-def delete_pets(pet_id):
+@app.route('/pets/<int:products_id>', methods=['DELETE'])
```
Comment (copatal8) [inline app/service.py]: change pets to products. Make products singular as product

File: app/service.py [hunk 2]
```diff
@@ -186,19 +186,19 @@ def update_pets(pet_id):
...
+def delete_products(products_id):
```
Comment (copatal8) [inline app/service.py]: change products_id to product id

File: app/service.py [hunk 3]
```diff
@@ -186,19 +186,19 @@ def update_pets(pet_id):
...
+    Delete a Product
 
-    This endpoint will delete a Pet based the id specified in the path
+    This endpoint will delete a Product based the id specified in the path
     """
-    app.logger.info('Request to delete pet with id: %s', pet_id)
-    pet = Pet.find(pet_id)
-    if pet:
-        pet.delete()
+    app.logger.info('Request to delete product with id: %s', products_id)
```
Comment (copatal8) [inline app/service.py]: product_id

File: app/service.py [hunk 4]
```diff
@@ -186,19 +186,19 @@ def update_pets(pet_id):
...
+    products = Products.find(products_id)
```
Comment (copatal8) [inline app/service.py]: product singular

File: app/service.py [hunk 5]
```diff
@@ -186,19 +186,19 @@ def update_pets(pet_id):
 
 
 ######################################################################
-# DELETE A PET
+# DELETE A PRODUCT
 ######################################################################
-@app.route('/pets/<int:pet_id>', methods=['DELETE'])
-def delete_pets(pet_id):
+@app.route('/products/<int:product_id>', methods=['DELETE'])
+def delete_product(product_id):
```
Comment (copatal8) [inline app/service.py:192]: delete_products(product_id)

**Review by copatal8 — APPROVED** [review]

**Pull request merged**